packages/react-textarea/src/stories/TextareaControlled.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Textarea } from '../Textarea';
+import { useId } from '@fluentui/react-utilities';
+import { Label } from '@fluentui/react-label';
+import type { TextareaProps } from '../Textarea';
+import { makeStyles, shorthands } from '@griffel/react';
+
+const useStyles = makeStyles({
```

Comment:
Small cleanup, but you could move this up to the Textarea.stories.tsx file so that all of the stories get the same wrapping div with styles.